
# HG changeset patch
# User Tassilo Philipp
# Date 1639763062 -3600
# Node ID 1e65b89116030845cbfd78c283ca681cda356e5a
# Parent  fde92442c7853ec00d96e7c62e3cbb7b21820fd4
- fix cmake build for test (missed to remove deleted file from cmakelists)

diff -r fde92442c785 -r 1e65b8911603 test/callback_suite/CMakeLists.txt
--- a/test/callback_suite/CMakeLists.txt	Fri Dec 17 11:47:18 2021 +0100
+++ b/test/callback_suite/CMakeLists.txt	Fri Dec 17 18:44:22 2021 +0100
@@ -1,3 +1,3 @@
-add_executable(callback_suite env.c handler.c sigstrings.c invokers.c do_test.c main.c print.c signature_utils.c)
+add_executable(callback_suite env.c handler.c sigstrings.c invokers.c do_test.c main.c signature_utils.c)
 target_link_libraries(callback_suite dyncallback_s)
 

